examples/src/keyring/multi/aws_kms_with_escrow.py

```diff
@@ -0,0 +1,125 @@
+by combining a KMS keyring with a raw RSA keyring.
+
+https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/choose-keyring.html#use-multi-keyring
+
+For more examples of how to use the KMS keyring, see the ``keyring/aws_kms`` examples.
+
+For more examples of how to use the raw RSA keyring, see the ``keyring/raw_rsa`` examples.
+
+In this example we generate a RSA keypair
+but in practice you would want to keep your private key in an HSM
+or other key management system.
+
+In this example, we use the one-step encrypt and decrypt APIs.
+"""
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.asymmetric import rsa
+
+import aws_encryption_sdk
+from aws_encryption_sdk.identifiers import WrappingAlgorithm
+from aws_encryption_sdk.keyrings.aws_kms import KmsKeyring
+from aws_encryption_sdk.keyrings.multi import MultiKeyring
+from aws_encryption_sdk.keyrings.raw import RawRSAKeyring
+
+
+def run(aws_kms_cmk, source_plaintext):
+    # type: (str, bytes) -> None
+    """Demonstrate an encrypt/decrypt cycle using a KMS keyring with a single CMK.
+
+    :param str aws_kms_cmk: The ARN of an AWS KMS CMK that protects data keys
+    :param bytes source_plaintext: Plaintext to encrypt
+    """
+    # Prepare your encryption context.
+    # https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/concepts.html#encryption-context
+    encryption_context = {
+        "encryption": "context",
+        "is not": "secret",
+        "but adds": "useful metadata",
+        "that can help you": "be confident that",
+        "the data you are handling": "is what you think it is",
+    }
+
+    # Generate an RSA private key to use with your keyring.
+    # In practice, you should get this key from a secure key management system.
+    #
+    # Why did we use this public exponent?
+    # https://latacora.singles/2018/04/03/cryptographic-right-answers.html
+    private_key = rsa.generate_private_key(public_exponent=65537, key_size=4096, backend=default_backend())
+
+    # Collect the public key from the private key.
+    public_key = private_key.public_key()
+
+    # Create the encrypt keyring that only has access to the public key.
+    escrow_encrypt_keyring = RawRSAKeyring(
+        # The key namespace and key name are defined by you
+        # and are used by the raw RSA keyring
+        # to determine whether it should attempt to decrypt
+        # an encrypted data key.
+        #
+        # https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/choose-keyring.html#use-raw-rsa-keyring
+        key_namespace="some managed raw keys",
+        key_name=b"my RSA wrapping key",
+        public_wrapping_key=public_key,
+        # The wrapping algorithm tells the raw RSA keyring
+        # how to use your wrapping key to encrypt data keys.
+        #
+        # Why did we use this wrapping algorithm?
+        # https://latacora.singles/2018/04/03/cryptographic-right-answers.html
```

Comment:
Replace this with just a comment that this this is our recommended padding and that PKCS #1v1.5 should not be used unless required for backwards compatibility.